Update query_program_input.py

vvkrddy opened the pull request:
Added NGC designation column for messier_objects.csv and a Messier column to NGC.csv where 1 means it's a messier and 0 means it's not a messier. 
Instead of overplotting, they could be skipped. 108 are matched.
But the main purpose of this pull request is I changed "TYPE' to 'Type'.

Commit:
Update query_program_input.py

Added NGC designation column for messier_objects.csv and a Messier column to NGC.csv where 1 means it's a messier and 0 means it's not a messier. 
Instead of overplotting, they could be skipped. 108 are matched.
But the main purpose of this pull request is I changed "TYPE' to 'Type'.

SahyadriDK commented on Program/query_program_input.py:
```diff
@@ -122,6 +133,19 @@
+        for j in range(len(x)):
+            if x[j] == result_table['Name'][i]:
+                result_table['Messier'][i]=1
+                bb.append(i)
```
Where is this list 'bb' used ultimately? Also would it be possible to reduce the number of nested for loops?

SahyadriDK reviewed (changes requested):
Made a comment please check.

Commit:
Update query_program_input.py

Removed 'bb' list; It wasn't meant to be in final code. I used it for verification.
Apparently the code is skipping two objects. I'll see what I can do. Let this be on hold.

vvkrddy commented:
Removed 'bb' list. It wasn't meant to go into final code.
Let this be on hold. I'll see what I can do with nested loop.
